.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+- id: swift-format
+  name: swift-format
+  entry: swift-format format --in-place --recursive --parallel
+  language: swift
+  types: [swift]
+  require_serial: true
+  
```

Comment:
The trailing newline has two spaces, could you remove them to make the YAML lint pass?

Reply:
Okay, done! 🤞